src/api/spec/controllers/concerns/triggerable_spec.rb

```diff
@@ -93,13 +103,27 @@ def stub_params(project_name:, package_name:)
       let(:token) { Token::Rebuild.create(user: user) }
       let(:project_with_a_link) { create(:project, name: 'project_with_a_link', maintainer: user, link_to: project) }
 
+      before do
+        fake_controller_instance.instance_variable_set(:@project_name, 'project_with_a_link')
+      end
+
       it 'authorizes the project if the package is from a project with a link' do
         stub_params(project_name: project_with_a_link.name, package_name: package.name)
         fake_controller_instance.set_project
         fake_controller_instance.set_package
         fake_controller_instance.set_object_to_authorize
         expect(fake_controller_instance.instance_variable_get(:@token).object_to_authorize).to eq(project_with_a_link)
       end
+    end
+
+    context 'project with project-link and a local package' do
```

Comment:
Why do we test this now? What changed?

Reply:
in this area nothing changed. I'm assuming @saraycp wanted to test the `project_name` and `package_name` instance variables?

Reply:
What's there to test? If instance variables don't work anymore we can go home anyway :)

Reply:
> What's there to test? If instance variables don't work anymore we can go home anyway :)

No, if using it changed some behavior in the controller, beside the other methods that were updated on Triggerable concern (`set_project` and `set_package`)

Reply:
The old tests should show this no?